test/e2e_node/gpu-device-plugin.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"fmt"
+	"os/exec"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+)
+
+const (
+	testPodNamePrefix = "nvidia-gpu-"
+	sleepTimeout      = 30
+)
+
+// Serial because the test restarts Kubelet
+var _ = framework.KubeDescribe("[Feature:GPUDevicePlugin][Serial]", func() {
```

Comment:
Do we need to have a separate yaml file like test/e2e_node/jenkins/gci-init-gpu.yaml?

Reply:
Never mind. I realized the test/e2e_node/jenkins/gci-init-gpu.yaml config is used by all tests running in serial.